Modules/posixmodule.c

```diff
@@ -5490,12 +5490,43 @@ os__path_normpath_impl(PyObject *module, PyObject *path)
         return NULL;
     }
     Py_ssize_t norm_len;
-    wchar_t *norm_path = _Py_normpath_and_size(buffer, len, &norm_len);
+    wchar_t *norm_path = _Py_normpath_and_size(buffer, len, 0, &norm_len);
     PyObject *result = PyUnicode_FromWideChar(norm_path, norm_len);
     PyMem_Free(buffer);
     return result;
 }
 
+/*[clinic input]
+os._path_abspath
+
+    path: object
+
+    start: Py_ssize_t=0
+
+Make path absolute.
```

Comment:
```suggestion
Make path absolute from character start.
```

Reply:
FTR, this docstring does not have any effect. See the import code in `Lib/posixpath.py` earlier in this PR.